tests/test_htmlparser.py

```diff
@@ -0,0 +1,31 @@
+        <meta name="geo.placename" content="Pays de la Loire" />
+        <meta name="DC.box" content="name=Pays de la Loire; northlimit=48.567994064425; southlimit=46.26666162307; westlimit=-2.6257394729017; eastlimit=0.91665065791138; projection=EPSG3857" />
+        <meta name="ISO 19139" content="<gmd:EX_GeographicBoundingBox><gmd:westBoundLongitude><gco:Decimal>-2.6257394729017</gco:Decimal></gmd:westBoundLongitude><gmd:eastBoundLongitude><gco:Decimal>0.91665065791138</gco:Decimal></gmd:eastBoundLongitude><gmd:southBoundLatitude><gco:Decimal>46.26666162307</gco:Decimal></gmd:southBoundLatitude><gmd:northBoundLatitude><gco:Decimal>48.567994064425</gco:Decimal></gmd:northBoundLatitude></gmd:EX_GeographicBoundingBox>" />
+        """
+        self.soup = BeautifulSoup(html_doc, 'html.parser')
+
+    def test_get_meta(self):
+        matching = self.soup("meta")
+        self.assertEqual(len(matching), 5)
+        self.assertEqual(matching[0].name, 'meta')
+        self.assertEqual(matching, self.soup.find_all("meta"))
+    
+    def test_get_geometry(self):
```

Comment:
Same here, you're not testing our code. Maybe this is just a temporary test?

Reply:
Updated two test files in the latest pull request Harvesting data from OJS #76 . Please check it out